Sources/SATSCore/Components/NoticeView/NoticeView.swift

```diff
@@ -20,6 +20,9 @@ public struct NoticeView: View {
         .padding(16)
         .background(notice.backgroundColor)
         .cornerRadius(8)
+        .shadow(color: Color.black.opacity(0.08), radius: 1, x: 0, y: 1)
+        .shadow(color: Color.black.opacity(0.12), radius: 5, x: 0, y: 1)
+        .shadow(color: Color.black.opacity(0.1), radius: 9, x: 0, y: 2)
```

Comment:
Customized shadow according to Figma

Reply:
how does this look in dark mode?

using concrete colors is not a good practice because of theming and dark mode

Reply:
![Skjermbilde 2022-01-27 kl  10 06 02](https://user-images.githubusercontent.com/9019310/151326982-3036ae42-9324-44c1-9ed2-c40d6683f497.png)

Just had a discussion with Magnus about shadowing. The conclusion for now is to have a color set with a black color in any appearance, and might change or add modifier (like SATSFont) later. This is updated now. @fespinoza